contracts/storage/tick.ral

```diff
@@ -50,10 +50,9 @@ Abstract Contract TickHelper() extends Decimal() {
             newLiquidity = tick.liquidityGross + liquidityDelta
         } else {
             newLiquidity = tick.liquidityGross - liquidityDelta
+            assert!(tick.liquidityGross >= liquidityDelta && newLiquidity < maxLiquidityPerTick, InvariantError.InvalidTickLiquidity)
```

Comment:
Are you sure it should be only in the `else` check?

Reply:
Good catch